renderer/src/pages/WalletConfig.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react'
+import FilAddressForm from '../components/FilAddressForm'
+import BackgroundGraph from './../assets/img/graph.svg'
+import { useNavigate } from 'react-router-dom'
+import { getFilAddress, setFilAddress } from '../lib/station-config'
+
+const WalletConfig = (): JSX.Element => {
+  const navigate = useNavigate()
+
+  const [docFilAddress, setDocFilAddress] = useState<string | undefined>(undefined)
```

Comment:
what does `docFilAddress` stand for?

Reply:
Stands for the address variable on current page. Changed it on 5d8078d

Reply:
nice going!